python_modules/dagster-graphql/dagster_graphql/schema/solids.py

```diff
@@ -446,6 +446,8 @@ def resolve_config_field(self, _graphene_info):
         )
 
     def resolve_required_resources(self, _graphene_info):
+        if not isinstance(self._solid_def_snap, SolidDefSnap):
```

Comment:
I think we should catch this at `__init__` time. The only other option is `CompositeSolidDefSnap`, and that appears to correspond to `GrapheneCompositeSolidDefinition`.

Reply:
good idea